src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-extension-card-base/sw-extension-card-base.html.twig

```diff
@@ -9,6 +9,15 @@
     <sw-loader v-if="isLoading" />
     {% endblock %}
 
+    <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+    {% block sw_extension_card_base_type_icon %}
+    <sw-icon
```

Comment:
what do you think about adding a tooltip that explains what the icon stands for?

Reply:
Good idea

Reply:
Done